resdk/resolwe.py

```diff
@@ -243,7 +243,7 @@ def _process_file_field(self, path):
 
     def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined-builtin
             descriptor_schema=None, collections=[],
-            data_name='', src=None, tools=None):
+            data_name='', src=None, tools=None, use_existing=False):
```

Comment:
And what to do if they don't match? Create new object with given parameters or raise an error? I don't like neither of these two options :)

Do you have any suggestions for better name?